src/core/api/chart_repository.go

```diff
@@ -296,30 +296,13 @@ func (cra *ChartRepositoryAPI) UploadChartVersion() {
 			cra.SendInternalServerError(err)
 			return
 		}
+		if err := cra.addEventContext(formFiles, cra.Ctx.Request); err != nil {
+			hlog.Infof("Failed to add chart upload context into request, which could lead to no execution of event based chart replication policy, %v", err)
```

Comment:
The event may be used by other non-replication components in the future, right? The message does not need to be so detailed.